public/input.css

```diff
@@ -2,6 +2,12 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer components {
+  .key {
+    @apply h-14 w-8 text-black rounded-xl border-2 border-gray-900 text-center text-2xl bg-spellie-blue-dark;
```

Comment:
Docs for creating component classes: https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply